assets/src/components/resource-view.scss

```diff
@@ -1,3 +1,15 @@
+div.resource-view {
```

Comment:
why do you need the "div" in front of the classname ? It should not be necessary and therefore not added.